manifests/testing/file-host.yaml

```diff
@@ -0,0 +1,71 @@
+---
+apiVersion: apps/v1beta2
+kind: Deployment
+metadata:
+  name: cdi-file-host
+  labels:
+    cdi.kubevirt.io: ""
+    cdi.kubevirt.io/testing: ""
+spec:
+  selector:
+    matchLabels:
+      cdi.kubevirt.io/testing: ""
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        cdi.kubevirt.io/testing: ""
+    spec:
+      initContainers:
+      - name: init
+        image: kubevirt/cdi-func-test-file-host-init
+        imagePullPolicy: IfNotPresent
+        args: ["-inFile", "/tmp/source/tinyCore.iso", "-outDir", "/tmp/shared"]
+        volumeMounts:
+        - name: "images"
+          mountPath: "/tmp/shared"
+      containers:
+      - name: http
+        image: kubevirt/cdi-func-test-file-host-http
+        imagePullPolicy: IfNotPresent
+        command: ["/usr/sbin/nginx"]
+        ports:
+        - name: http
+          containerPort: 80
+        volumeMounts:
+        - name: "images"
+          mountPath: "/tmp/shared/"
+      - name: s3
+        image: minio/minio:RELEASE.2018-08-02T23-11-36Z
+        imagePullPolicy: IfNotPresent
+        env:
+        - name: MINIO_ACCESS_KEY
+          value: "admin"
+        - name: MINIO_SECRETY_KEY
+          value: "password"
+        args: ["server", "--address", ":9000", "/tmp/shared"]
+        volumeMounts:
+        - name: "images"
+          mountPath: "/tmp/shared"
+      volumes:
+      - name: "images"
+        emptyDir: {}
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: cdi-file-server
+  labels:
+    cdi.kubevirt.io: ""
+    cdi.kubevirt.io/testing: ""
+spec:
+  selector:
+      cdi.kubevirt.io/testing: ""
+  type: NodePort
```

Comment:
why NodePort here instead of the default ClusterIP type?

Reply:
I would take it from ingress deployment of nginx, but perhaps this is a bad example: https://github.com/kubernetes/ingress-nginx/blob/master/deploy/provider/baremetal/service-nodeport.yaml

Reply:
we don't need ingress in for the http/s3 file servers though. The only thing that are accessing these servers are other pods within the cluster. Access for our tests will all be self contained within the cluster's pod network.

Reply:
@davidvossel 
> why NodePort here instead of the default ClusterIP type?

Agreed - this is part of the scratch yaml that I'll be fixing up today.  NodePort allowed me to quickly debug requests to the http server with `curl`.  Since the file server will only be responding to in-cluster requests, it will be changed to ClusterIP